src/akamai/js/akamaiHD_flash.js

```diff
@@ -98,14 +101,14 @@ require("../../../html5-common/js/utils/constants.js");
      * Creates a video player instance using OoyalaAkamaiHDFlashVideoWrapper.
      * @public
      * @method OoyalaAkamaiHDFlashVideoFactory#create
-     * @param {object} parentContainer The jquery div that should act as the parent for the video element
+     * @param {video} parentContainer The jquery div that should act as the parent for the video element
      * @param {string} id The id of the video player instance to create
      * @param {object} controller A reference to the video controller in the Ooyala player
      * @param {object} css The css to apply to the video element
      * @returns {object} A reference to the wrapper for the newly created element
      */
     this.create = function(parentContainer, id, controller, css) {
-      var video = $("<object>");
+      var video = $("<video>");
```

Comment:
I think we had talked about this in the past.  Since this element is not used as a video element, why are we creating it as one?  We shouldn't create a video element unless we intend to use it as such